go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/flyandi/twiml
```

Comment:
since this is being merged into BTBurke/twiml, you'll probably want to change this here.